grails-app/views/inventory/list.gsp

```diff
@@ -56,8 +56,8 @@
                                 <th class="center"><warehouse:message code="inventoryLevel.status.label" default="Status"/></th>
                                 <th><warehouse:message code="product.productCode.label" default="Code"/></th>
                                 <th><warehouse:message code="product.label" default="Product"/></th>
+                                <th class="left"><warehouse:message code="product.productFamily.label" default="ProductFamily"/></th>
```

Comment:
shouldn't there be a space between `ProductFamily` => `Product Family` in the default label?

Reply:
you are right, thanks.